examples/kotlin/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://s.apache.org/beam-owners
```

Comment:
In a follow-up, I think we should just remove OWNERS here. I don't think it is really used for anything and this is not a large enough or accurate list or who to reach out for help